src/statement_generator.cpp

```diff
@@ -1120,9 +1139,8 @@ unique_ptr<ParsedExpression> StatementGenerator::GenerateSubquery() {
 		return GenerateConstant();
 	}
 	auto subquery = make_uniq<SubqueryExpression>();
-
 	{
-		StatementGenerator child_generator(*this);
+		StatementGenerator child_generator(*this, verification_enabled);
```

Comment:
Is it not possible to change the StatementGenerator constructor to look like so? In src/statement_generator.cpp?

```c++
StatementGenerator::StatementGenerator(StatementGenerator &parent_p)
    : context(parent_p.context), parent(&parent_p), generator_context(parent_p.generator_context),
      depth(parent_p.depth + 1), enable_verification(parent_p.enable_verification) {
	if (depth > MAX_DEPTH) {
		throw InternalException("depth too high");
	}
}
```

Reply:
Yes, that's way it looks better! Thanks!